web/angular-wallet/src/app/disclaimer/disclaimer.component.ts

```diff
@@ -43,12 +43,10 @@ export class DisclaimerComponent
       try {
 
         const settings = await this.storeService.getSettings();
-        await this.storeService.saveSettings({
-          ...settings,
-          agree: true
-        });
+        settings.agree = true;
```

Comment:
this is how lokijs really works, and how we should use the store (or, I'm terribly wrong - which is also possible hehe)

Reply:
it prefers mutation? hmm

Reply:
`update` from loki works this way (and btw every ORM, like hibernate, entity framework, etc.). It keeps track of persisted objects, and so partial updates work like this. no need to reconstruct entire objects. IMHO easier